task_schedules/task_schedule_supplement_dispositions.cfg

```diff
@@ -0,0 +1,55 @@
+# Configuration file for task_schedule.pl to update the AGASC supplement
+
+subject           agasc supplement observation disposition
+timeout           86400             # Default tool timeout
+heartbeat_timeout 1                 # Maximum age of heartbeat file (seconds)
+iterations        1                 # Run once then shut down task_schedule
+print_error       1                 # Print full log of errors
+disable_alerts    0                 # Don't disable alerts since this jobs runs just once/day
+loud              0                 # Run loudly or quietly (production mode)
+
+# Data files and directories.  The *_dir vars can have $ENV{} vars which
+# get interpolated.  (Note lack of task name after TST_DATA because this is just for test).
+
+data_dir     $ENV{SKA}/data/agasc/rc/tasks/disposition       # Data file directory
+log_dir      $ENV{SKA}/data/agasc/rc/logs  # Log file directory
```

Comment:
I think this can be `$ENV{SKA}/data/agasc/logs`. We normally put processing logs at the top level like that.

Reply:
Done.